contracts/src/v0.8/functions/dev/v1_X/FunctionsBilling.sol

```diff
@@ -282,19 +341,22 @@ abstract contract FunctionsBilling is Routable, IFunctionsBilling {
     ) {
       delete s_requestCommitments[requestId];
       // Reimburse the transmitter for the fulfillment gas cost
-      s_withdrawableTokens[msg.sender] = gasOverheadJuels + callbackCostJuels;
+      s_withdrawableTokens[msg.sender] += gasOverheadJuels + callbackCostJuels;
       // Put donFee into the pool of fees, to be split later
       // Saves on storage writes that would otherwise be charged to the user
       s_feePool += commitment.donFee;
+      // Pay the operation fee to the Coordinator owner
+      s_withdrawableTokens[_owner()] += commitment.operationFee;
```

Comment:
So fees will be held by the EngOps multisig wallet?

Reply:
On mainnets the current owner of the Coordinator is the _Chainlink Labs_ multisig. Eng Ops just creates multisig proposals.

I'll be really verbose for completeness: The actual LINK ERC20 ownership is held by the Router. The Router has its own book keeping and assigns a lump withdrawable amount to the Coordinator on every request. The Coordinator contract also does its own book keeping for who is owed what pieces of the lump amount (seen here on line 349).

Reply:
Thanks. I'll give the full withdrawal logic another review when you are ready for the next review iteration on this PR.